evalml/tests/component_tests/test_stacked_ensemble_regressor.py

```diff
@@ -4,191 +4,232 @@
-def test_stacked_feature_importance(mock_fit, X_y_regression, stackable_regressors):
+@patch("evalml.pipelines.components.ensemble.StackedEnsembleRegressor.fit")
+def test_stacked_feature_importance(mock_fit, X_y_regression):
     X, y = X_y_regression
-    input_pipelines = [
-        RegressionPipeline([regressor]) for regressor in stackable_regressors
-    ]
-    clf = SklearnStackedEnsembleRegressor(input_pipelines=input_pipelines, n_jobs=1)
+    clf = StackedEnsembleRegressor(n_jobs=1)
     clf.fit(X, y)
     mock_fit.assert_called()
     clf._is_fitted = True
     with pytest.raises(
-        NotImplementedError, match="feature_importance is not implemented"
+        NotImplementedError,
+        match="feature_importance is not implemented for StackedEnsembleClassifier and StackedEnsembleRegressor",
     ):
         clf.feature_importance
+
+
+def test_stacked_same_model_family():
+    graph_en = {
+        "Imputer": ["Imputer", "X", "y"],
+        "Target Imputer": ["Target Imputer", "X", "y"],
+        "OneHot_RandomForest": ["One Hot Encoder", "Imputer.x", "Target Imputer.y"],
+        "OneHot_ElasticNet": ["One Hot Encoder", "Imputer.x", "y"],
+        "Random Forest": ["Random Forest Regressor", "OneHot_RandomForest.x", "y"],
+        "Elastic Net": ["Elastic Net Regressor", "OneHot_ElasticNet.x", "y"],
+        "EN": [
+            "Elastic Net Regressor",
+            "Random Forest.x",
+            "Elastic Net.x",
+            "y",
+        ],
+    }
+
+    graph_rf = {
+        "Imputer": ["Imputer", "X", "y"],
+        "Target Imputer": ["Target Imputer", "X", "y"],
+        "OneHot_RandomForest": ["One Hot Encoder", "Imputer.x", "Target Imputer.y"],
+        "OneHot_ElasticNet": ["One Hot Encoder", "Imputer.x", "y"],
+        "Random Forest": ["Random Forest Regressor", "OneHot_RandomForest.x", "y"],
+        "Elastic Net": ["Elastic Net Regressor", "OneHot_ElasticNet.x", "y"],
+        "Linear Regressor": [
+            "Linear Regressor",
+            "Random Forest.x",
+            "Elastic Net.x",
+            "y",
+        ],
+    }
+
+    RegressionPipeline(component_graph=graph_en)
+    RegressionPipeline(component_graph=graph_rf)
+
+    input_pipelines = [
+        RegressionPipeline(component_graph=graph_en),
+        RegressionPipeline(component_graph=graph_rf),
+    ]
+
+    pipeline = _make_stacked_ensemble_pipeline(
+        input_pipelines=input_pipelines,
+        problem_type=ProblemTypes.REGRESSION,
+        use_sklearn=False,
+    )
+
+    assert "Linear Pipeline - Imputer" in pipeline.component_graph.compute_order
+    assert "Linear Pipeline 2 - Imputer" in pipeline.component_graph.compute_order
+
+    assert "Linear Pipeline - Elastic Net" in pipeline.component_graph.compute_order
+    assert "Linear Pipeline 2 - Elastic Net" in pipeline.component_graph.compute_order
+    assert "Linear Pipeline - EN" in pipeline.component_graph.compute_order
+
+
+def test_automl_ensembler_str_and_classes():
```

Comment:
Probably don't need automl in this name? Even though it seems a little silly, maybe we should have a stacked ensemble classifier with this test too! (aka mirror tests in regression / classification when possible)